UnityProject/Assets/Scripts/UI/Core/Animations/AnimatedRadialRotation.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using UI.Core.Radial;
+using UnityEngine;
+
```

Comment:
add a namespace for any new script you add. In this case the namespace would be ``Core.Animations``